flags/urls.py

```diff
@@ -127,7 +128,7 @@ def _flagged_path(
     elif isinstance(view, (list, tuple)):
         urlconf_module, app_name, namespace = view
 
-        route_pattern = Pattern(route, name=name, is_endpoint=True)
+        route_pattern = Pattern(route, name=name, is_endpoint=False)
```

Comment:
This function is a mirror of [Django's `_path` function](https://github.com/django/django/blob/main/django/urls/conf.py#L68-L78), which sets `is_endpoint` to `False` for `include` patterns (anything that match `(list, tuple)`). This change aligns this with Django's, and its misalignment is probably a mistake that's been here since this code was written, but probably didn't show up until Django 2.0 when `path()` became an option.